sensor/tests/resource/helper.go

```diff
@@ -54,6 +54,18 @@ type YamlTestFile struct {
 	File string
 }
 
+// awaitedResources slice of resources that need to be awaited
```

Comment:
nit: `awaited` seems like the wrong word here. Maybe `waited` or `required` would fit better what you mean.

Reply:
I went with `required` for some reason `waited` doesn't sound good to me

Reply:
Done